### PR TITLE
Update MODS mappings

### PR DESCRIPTION
**Pull request by no-reply**
Fixes `originInfo`, and case sensitivity for role terms

### CODE REVIEW

File: heidrun/ncdhc.rb [hunk 1]
```diff
@@ -33,24 +33,28 @@
...
     		:as => :contrib do
       providedLabel contrib.field('mods:namePart')
     end
 
     creator :class => DPLA::MAP::Agent, 
     	    :each => record.field('mods:name')
-    		    .select { |name| name['mods:role'].map(&:value).include?('creator') }, 
+    		    .select { |name| name['mods:role']
+                              .map { |v| v.value.downcase }
+    		              .include?('creator') },
```
Comment (guegueng) [inline heidrun/ncdhc.rb]: I think this might be a little indentation error

Comment (no-reply) [inline heidrun/ncdhc.rb]: This is odd. If I check this out, these are indented the same in my editor's `ruby-mode`. Can't figure out what's going on here. There shouldn't be whitespace issues.

Comment (no-reply) [inline heidrun/ncdhc.rb]: Okay, got this cleared up.

**Comment by no-reply** [comment]
whitespace diffs +-

**Pull request merged**